src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonDictionaryConverter.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Text.Json.Serialization
     /// </summary>
     internal abstract class JsonDictionaryConverter<T> : JsonResumableConverter<T>
     {
-        internal override ClassType ClassType => ClassType.Dictionary;
+        internal sealed override ClassType ClassType => ClassType.Dictionary;
```

Comment:
We should seal the overrides in `JsonCollectionConverter` and `JsonObjectConverter` as well.

Reply:
fixed in 02ab96b